### ISSUE TITLE
Real time compressor and limiter: accessibility names of sliders

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#6475]
### Bug description

In the real time compressor and limiter it would be less confusing if the accessibility names of the sliders were just the names of the properties, rather than also including the units, eg, just Threshold, Knee width, etc. For these sliders, screen readers read a percentage value between 0 and 100. If units are included in the accessibility name, eg db, then the user may mistakenly assume that the percentage value is the value in db.

Compare this with, for example, Audacity's Bass and Treble effect, where the accessibility names of the sliders are just the names of the properties.

### Steps to reproduce

With a screen reader running:
1. Select some audio.
2. On the effects menu, open either Audacity's limiter or compressor.
3. Tab to the slider for knee width.

### Expected behavior

The name of the slider read by the screen reader is "knee width".

### Actual behavior

The name of the slider read by the screen reader is "knee width (db)"

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment]
Does this only apply to the limiter and compressor?  I'm asking because the name/unit/textinput/slider combo is something that's pretty common in Audacity - including the old limiter. 

![image](https://github.com/audacity/audacity/assets/87814144/44c15360-ad05-4be4-b049-07b8a9f016bf)

It's also a thing in Noise Reduction, Tremolo, Delay, Distortion and probably more.

**Comment by DavidBailes** [comment]
> Does this only apply to the limiter and compressor? I'm asking because the name/unit/textinput/slider combo is something that's pretty common in Audacity - including the old limiter.

No, some existing effects have this issue, but others don't. For example, in the Distortion effect, then the issue exists for the clipping level, whilst for the Noise reduction effect, the noise reduction parameter doesn't have this issue.
For all nyquist effects (like the old limiter), the sliders have no names, since there is currently no way of setting custom accessibility names in this case.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
